dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESMappingAPIImpl.java

```diff
@@ -304,8 +313,28 @@ public String toJson(Contentlet con) throws DotMappingException {
 		}
 	}
 
+    /**
+     * Set the Vanity Url URI in the content map
+     * @param con The current vanity url Contentlet
+     * @param contentletMap Contentlet properties map
+     * @param ident Contentlet
+     */
+    private void populateVanityUrlPath(Contentlet con, Map<String, String> contentletMap, Identifier ident) {
+        String vanityUrlPath;
+        try{
+            VanityUrl vanityUrl = APILocator.getVanityUrlAPI().getVanityUrlFromContentlet(con);
+            vanityUrlPath = VanityUrlUtil.fixURI(vanityUrl.getURI());
+            if(vanityUrlPath != null){
+                contentletMap.put(ESMappingConstants.VANITY_URL,vanityUrlPath);
+            }
+        }catch(Exception e){
+            Logger.warn(this.getClass(), "Cannot get Vanity URL for contentlet.id : " + ((ident != null) ? ident.getId() : con) + " , reason: "+e.getMessage());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 160 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)